config/routes.rb

```diff
@@ -87,6 +87,7 @@
         edit_step :contact_details
         edit_step :office_code
         edit_step :case_details
+        edit_step :case_category
```

Comment:
Nothing stopping someone from accessing this early, should be a constrained route.

Reply:
agreed - but seen what the feature flag is now so will add the code in based on the fact that I've already seen what Robin is adding for this